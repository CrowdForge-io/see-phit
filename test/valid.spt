R"*(
<spaan>
  <p  color="red" height='1' weight="test" > 
    Extra whitespace after close and open tags
  </p  >
  
  <DIV id="hello"> 
    Extra space after void tag
  </DIV>
  <IMG />

  <DIV> 
    <DIV> 
      Nested tag, void tag
    </DIV>
    <IMG/>
  </DIV>
  
  <PI>Inline tag</PI>
  <Span>Mixed case tag name</SPAN>

</spaan>

<DIV>
  <DIV> 
    Multiple top level nodes
  </DIV>
</DIV>
)*"_html; 