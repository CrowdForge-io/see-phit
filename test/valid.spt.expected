Warning() [with int ROW = 2; int COL = 2; WHAT = spt::Unknown_tag_name]':
Warning() [with int ROW = 19; int COL = 4; WHAT = spt::Unknown_tag_name]':
